src/companion-connection.ts

```diff
@@ -109,12 +123,16 @@ class CompanionConnection extends EventEmitter<CompanionConnectionEvents> {
 
 	connect() {
 		console.log('cc: connect')
-		const websocket = (this.#websocket = new WebSocket('ws://' + this.address + ':28492'))
+		streamDeck.logger.info('Connecting to Companion at', this.address, this.port)
+		const websocket = (this.#websocket = new WebSocket('ws://' + this.address + ':' + this.port))
```

Comment:
It feels like there should be some check and cleanup of the existing value of `this.#websocket` here. Not a new issue, but could be a more likely leak?